examples/sycl/pvc/flash_attention_v2/pvc_flash_attn_runner.hpp

```diff
@@ -31,10 +31,10 @@
 #pragma once
 
 #include "cutlass/epilogue/collective/default_epilogue.hpp"
-#include "cutlass/epilogue/fusion/xe_callbacks.hpp"
 #include "cutlass/gemm/device/gemm_universal_adapter.h"
-#include "pvc_flash_attn_gemm_universal.hpp"
-#include "pvc_flash_attn_epilogue.hpp"
+#include "../applications/pvc/flash_attention_v2/kernel/pvc_flash_attn_gemm_universal.hpp"
+#include "../applications/pvc/flash_attention_v2/collective/pvc_flash_attn_epilogue.hpp"
+#include "../applications/pvc/flash_attention_v2/collective/pvc_softmax_epilogue.hpp"
```

Comment:
```suggestion
#include "applications/pvc/flash_attention_v2/kernel/pvc_flash_attn_gemm_universal.hpp"
#include "applications/pvc/flash_attention_v2/collective/pvc_flash_attn_epilogue.hpp"
#include "applications/pvc/flash_attention_v2/collective/pvc_softmax_epilogue.hpp"
```
do you need the `../`, since `applications` is a top level directory? I guess you might need to add it to the include path

Reply:
Fixed this.

Reply:
Seems a little off actually.
Maybe you should move the files to `applications/include/applications/pvc/flash...` if this code is supposed to be included. I can't decide if this is standard or weird :laughing:. What do you think?

Reply:
Yeah, I have changed the code again to remove the `pvc` folder and renamed the files to show that this code can run on other devices Intel as well. Adding more folders wasn't really going to work in my opinion.
Now the folder hierarchy makes more sense to me since it holds an application (flash_attention_v2) inside it and the files' naming is similar to what we have inside `cutlass/gemm/collective|kernel` and `cutlass/epilogue/collective` with the idea being that each application can have it's own implementation of the relevant cutlass components within it's specific directory.